sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java

```diff
@@ -11,71 +10,103 @@
-        //Send an sms to only one phone number
-        SmsSendOptions options = new SmsSendOptions();
-        options.setDeliveryReportEnabled(true);
-        options.setTag("Tag"); /* Optional */
-        // Send the message to a list of  phone Numbers and check the response for a messages ids
-        SmsSendResult response = smsClient.send(
+    public SmsClient createSmsClientWithAAD() {
+        // You can find your endpoint and access key from your resource in the Azure Portal
+        String endpoint = "https://<RESOURCE_NAME>.communication.azure.com";
+
+        // Create an HttpClient builder of your choice and customize it
+        HttpClient httpClient = new NettyAsyncHttpClientBuilder().build();
+
+        SmsClient smsClient = new SmsClientBuilder()
+            .endpoint(endpoint)
+            .credential(new DefaultAzureCredentialBuilder().build())
+            .httpClient(httpClient)
+            .buildClient();
+
+        return smsClient;
+    }
+
+    public void sendMessageToOneRecipient() {
+        SmsClient smsClient = createSmsClientUsingAzureKeyCredential();
+
+        SmsSendResult sendResult = smsClient.send(
             "<from-phone-number>",
             "<to-phone-number>",
-            "your message",
-            options /* Optional */);
-        System.out.println("MessageId: " + response.getMessageId());
+            "Hi");
+
+        System.out.println("Message Id: " + sendResult.getMessageId());
+        System.out.println("Send Result Successful:" + sendResult.isSuccessful());
     }
 
-    public void sendMessageToMultipleRecipients(SmsClient smsClient) {
-        //Send an sms to multiple phone numbers
+    public void sendMessageToGroupWithOptions() {
+        SmsClient smsClient = createSmsClientUsingAzureKeyCredential();
+
         SmsSendOptions options = new SmsSendOptions();
         options.setDeliveryReportEnabled(true);
-        options.setTag("Tag"); /* Optional */
-        // Send the message to a list of  phone Numbers and check the response for a messages ids
-        Iterable<SmsSendResult> responseMultiplePhones = smsClient.send(
+        options.setTag("Tag");
+
+        Iterable<SmsSendResult> sendResults = smsClient.sendWithResponse(
             "<from-phone-number>",
-            new ArrayList<String>(Arrays.asList("<to-phone-number1>", "<to-phone-number2>")),
-            "your message",
+            Arrays.asList("<to-phone-number1>", "<to-phone-number2>"),
+            "Hi",
             options /* Optional */,
-            null);
-        for (SmsSendResult messageResponseItem : responseMultiplePhones) {
-            System.out.println("MessageId sent to " + messageResponseItem.getTo() + ": " + messageResponseItem.getMessageId());
+            Context.NONE).getValue();
+
+        for (SmsSendResult result : sendResults) {
+            System.out.println("Message Id: " + result.getMessageId());
+            System.out.println("Send Result Successful:" + result.isSuccessful());
```

Comment:
Here I think will be nice to show the developer to how to check to whom the messge was sent like System.out.println(" Message sent to : " result.getTo());